CWE/events/romania.txt

```diff
@@ -1732,13 +1798,20 @@ country_event = {
 		
 		#Expensive projects
 		OR = { 
-			has_country_flag = systematization_ROM
 			has_country_flag = invested_in_electronics
 			has_country_flag = ROM_western_path_flag
+
 		}
-		has_country_flag = palace_of_the_parliament_ROM_flag
-		has_country_flag = romanian_cultural_revolution
-		has_country_flag = decree_770_flag
+		OR = {
+			has_country_flag = palace_of_the_parliament_ROM_flag
+			has_country_flag = systematization_ROM
+
```

Comment:
Might be good to insert these into this OR bracket so that any attempt to rebuild budapest, through generic or historic events can also cause austerity.
capital_scope = { has_province_modifier = new_capital }
has_country_flag = germania
has_country_modifier = germania

Reply:
Alright.